Dockerfile-csi-test

```diff
@@ -15,6 +15,7 @@
 FROM golang:1.11
 
 ARG CSI_PARAMS=csi_params
+ARG STORAGE_LABEL=community_svc
```

Comment:
could you remove the `community_` prefix? it should be implied from the context.
also I think in the project `STORAGE_LABEL` would be called `ARRAY_TYPE`